bin/dev/add_protoc_plugin.sh

```diff
@@ -0,0 +1 @@
+asdf plugin add protoc https://github.com/paxosglobal/asdf-protoc.git
```

Comment:
probably an argument to be made that this script should be renamed and also run `asdf install protoc`